platform-sdk/platform-apps/tests/StatsSigningTestingTool/src/main/java/com/swirlds/demo/stats/signing/StatsSigningTestingToolState.java

```diff
@@ -127,93 +101,36 @@ public void handleConsensusRound(final Round round, final SwirldDualState swirld
     }
 
     private void handleTransaction(final ConsensusTransaction trans) {
-        for (final TransactionSignature s : trans.getSignatures()) {
-
-            if (!validateSignature(s, trans)) {
```

Comment:
Did you check with Nathan to see if there is any value in keeping this transaction signing stuff around? I was not under the impression this would be removed.